core/iwasm/aot/aot_intrinsic.c

```diff
@@ -423,6 +427,9 @@ int32
 aot_intrinsic_f32_cmp(AOTFloatCond cond, float32 lhs, float32 rhs)
 {
     switch (cond) {
+        case FLOAT_EQ:
+            return fabsf(lhs - rhs) <= __FLT_EPSILON__ ? 1 : 0;
```

Comment:
fabsf isn't supported in zephyr platform, in fact we implemented fabs for in with core/shared/platform/common/math/math.c which is ported from open sourced math lib.
Could we use fabs to implement the fabsf, like in interpreter: `use (float32)fabs(lhs - rhs)` instead